src/test/java/com/wiley/autotest/framework/tests/repeater/RepeaterTest.java

```diff
@@ -3,23 +3,21 @@
 import com.wiley.autotest.framework.config.BaseTest;
 import com.wiley.autotest.framework.pages.RepeaterPage;
 import com.wiley.autotest.services.StopTestExecutionException;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.testng.annotations.Test;
 
 public class RepeaterTest extends BaseTest {
 
-    @Autowired
-    private RepeaterPage page;
-
-    @Test(expectedExceptions = StopTestExecutionException.class)
-    public void unableToPerformAction() {
+    @Test()
+    public void performAction() {
         openPage("repeater.html");
-        page.negativeCheck();
+        getPage(RepeaterPage.class)
```

Comment:
what if we simplify a bit more. Create getPage method in BaseTest and add "openPage" to it. 
getPage(htmlFileName, pageClass)?